Automattic/wp-calypso
# Social Login: Allow logging into an existing WP.com account with a Google account

Sadly, #15543 was reverted, due to the `wpcom-proxy-request` upgrade causing errors on non-rest responses.

This PR reverts #16427.

#### Testing instructions

(I haven't been able to reproduce this, yet - I'll update the testing instructions when I figure it out.)

0. Apply server side patch D6499 and sandbox `public-api.wordpress.com`
1. go to a site with posts
2. click on ‘share’
3. schedule a share to facebook
4. see an error message, even though you get a 200 response back

#### Reviews
